lib/compare-results/index.js

```diff
@@ -40,10 +34,6 @@ async function main() {
 
     const [masterTests, prTests] = await Promise.all([parseTestResults(masterArtifactFilePath), parseTestResults(prArtifactFilePath)]);
 
-    if (masterTests.length !== prTests.length) {
```

Comment:
Could we filter out the intermittent tests before comparing the lengths? IMO this check is necessary if a pull request is basing on an old master branch and we can remind developer to rebase.

Reply:
It might help to turn on this check:
<img width="718" alt="Screen Shot 2019-11-15 at 5 11 15 PM" src="https://user-images.githubusercontent.com/2481105/68979227-f6957680-07ca-11ea-9e15-25fb1192b1ad.png">

Reply:
I can leave the check, but my main concern is checking the same number of test for this purpose is not very definitive as it could be different for different changes to intermittent-tests.txt